clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -36,7 +39,7 @@
     implements ServiceCreationConfiguration<ClusteringService>,
     CacheManagerConfiguration<PersistentCacheManager> {
 
-  private static final String CLUSTER_SCHEME = "terracotta";
+  private static final Collection<String> CLUSTER_SCHEMES = new HashSet<String>(Arrays.asList("terracotta", "passthrough"));
```

Comment:
Wouldn't a test-module subclass of `ClusteringServiceConfiguration` that allows the `passthrough` scheme would do the job?